test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreLoggingTests.cs

```diff
@@ -139,6 +140,77 @@ public async Task RestoreLogging_VerifyNU1608MessageAsync()
                 log.Message.Should().Contain("Detected package version outside of dependency constraint: x 1.0.0 requires z (= 1.0.0) but version z 2.0.0 was resolved.");
             }
         }
+        [Fact]
+        public async Task RestoreLogging_VerifyNU5036WithMissingNuspecInSourceErrorMessageAsync()
```

Comment:
naming: Consider  following the above pattern. 
Makes test names more readable imo. 

I get if you want to keep it though, since it will match the pattern in this class.